clang/include/clang/CConv/CheckedRegions.h

```diff
@@ -13,78 +13,87 @@
 #ifndef _CHECKEDREGIONS_H
 #define _CHECKEDREGIONS_H
 
+#include <utility>
+
 #include "clang/AST/Decl.h"
 #include "clang/AST/Stmt.h"
 #include "clang/AST/ASTContext.h"
 #include "clang/Rewrite/Core/Rewriter.h"
 
 #include "ProgramInfo.h"
 
-using namespace clang;
```

Comment:
This is causing you to add `clang::` in a lot of places. Why was it necessary to drop this?

Reply:
Style guides I read suggest dropping `using namespace` in headers as anyone who imports the header is now also importing the namespace. `using namespace` is still used in the source file

Reply:
OK.